### PR TITLE
Added function to delete chat on the server. Updated spanish localiza…

### PR DESCRIPTION
**Pull request by scmanjarrez**
…tion. Closes #540

### CODE REVIEW

**Comment by scmanjarrez** [comment]
Related #540

File: src/EdgeGPT/chathub.py [hunk 1]
```diff
@@ -229,5 +229,22 @@ async def _initial_handshake(self, wss: aiohttp.ClientWebSocketResponse) -> None
         await wss.receive(timeout=900)
         await wss.send_str(append_identifier({"type": 6}))
 
+    async def remove_and_close(self) -> None:
```
Comment (acheong08) [inline src/EdgeGPT/chathub.py:232]: Would prefer to have this as `remove_conversation(conversation_id:str=None, conversation_signature:str=None,client_id:str=None)`

File: src/EdgeGPT/chathub.py [hunk 2]
```diff
@@ -229,5 +229,22 @@ async def _initial_handshake(self, wss: aiohttp.ClientWebSocketResponse) -> None
...
+        conversation_id = self.request.conversation_id
+        conversation_signature = self.request.conversation_signature
+        client_id = self.request.client_id
```
Comment (acheong08) [inline src/EdgeGPT/chathub.py:235]: `conversation_id = conversation_id or self.request.conversation_id`

**Review by acheong08 — CHANGES_REQUESTED** [review]

**Comment by acheong08** [comment]
Same for the Chatbot class

File: src/EdgeGPT/chathub.py [hunk 3]
```diff
@@ -229,5 +229,22 @@ async def _initial_handshake(self, wss: aiohttp.ClientWebSocketResponse) -> None
...
+            url,
+            json={
+                "conversationId": conversation_id,
+                "conversationSignature": conversation_signature,
+                "participant": {"id": client_id},
+                "source": "cib",
+                "optionsSets": ["autosave"]
+            }
+        )
+        await self.session.aclose()
```
Comment (acheong08) [inline src/EdgeGPT/chathub.py:247]: And this removed. Should be separate from close

**Comment by scmanjarrez** [comment]
Sorry, I've been pretty busy. Couldn't do the suggested changes

**Comment by acheong08** [comment]
No worries. I resolved them in https://github.com/acheong08/EdgeGPT/commit/b08633010b2b0db3ae58a1fcc098f7846906f5eb

**Pull request merged**